eng/common/pipelines/templates/steps/docs-metadata-release.yml

```diff
@@ -43,16 +33,21 @@ steps:
   displayName: 'Apply Documentation Updates From Artifact'
   inputs:
     targetType: filePath
-    filePath: ${{ parameters.ScriptDirectory }}/release-docs-metadata.ps1
-    arguments: -ArtifactLocation ${{parameters.ArtifactLocation}} -Repository ${{parameters.PackageRepository}} -ReleaseSHA ${{parameters.ReleaseSha}} -RepoId ${{parameters.RepoId}} -WorkDirectory '${{parameters.WorkingDirectory}}' -DocRepoLocation "${{parameters.WorkingDirectory}}/repo" -Language "${{parameters.Language}}" -DocRepoContentLocation "docs-ref-services/"
+    filePath: ${{ parameters.ScriptDirectory }}/update-docs-metadata.ps1
+    arguments: -ArtifactLocation ${{parameters.ArtifactLocation}} -Repository ${{parameters.PackageRepository}} -ReleaseSHA ${{parameters.ReleaseSha}} -RepoId ${{parameters.RepoId}} -WorkDirectory '${{parameters.WorkingDirectory}}' -DocRepoLocation "${{parameters.WorkingDirectory}}/repo" -Language "${{parameters.Language}}" -DocRepoContentLocation ${{ parameters.DocRepoDestinationPath }}
     pwsh: true
   timeoutInMinutes: 5
   env:
     GH_TOKEN: $(azuresdk-github-pat)
 
 - template: /eng/common/pipelines/templates/steps/create-pull-request.yml
   parameters:
-    RepoName: ${{ parameters.TargetDocRepo }}
-    PRBranchName: increment-package-version-${{ parameters.ArtifactName }}-$(Build.BuildId)
+    RepoName: ${{ parameters.TargetDocRepoName }}
+    RepoOwner: ${{ parameters.TargetDocRepoOwner }}
+    PRBranchName: ${{ parameters.PRBranchName }}
     CommitMsg: "Increment package version after release of ${{ parameters.ArtifactName }}"
```

Comment:
I assume you want to update the commit message as well.

Reply:
On my local it _is_. 

![image](https://user-images.githubusercontent.com/45376673/81213696-99b19680-8f8b-11ea-816e-423f17600136.png)

Looking.